packages/calcite-design-tokens/package.json

```diff
@@ -12,17 +12,21 @@
   "type": "module",
   "exports": {
     "./css": "./dist/css/calcite-headless.css",
-    "./css/brand/dark": "./dist/css/brand-dark.css",
-    "./css/brand/light": "./dist/css/brand-light.css",
     "./css/calcite/dark": "./dist/css/calcite-dark.css",
     "./css/calcite/light": "./dist/css/calcite-light.css",
-    "./css/headless": "./dist/css/calcite-headless.css",
```

Comment:
Should we keep `./css/headless` for backwards compatibility?

Reply:
yep! sorry I pulled from my November release branch.